php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S104.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Files should not have too many lines",
+  "title": "Files should not have too many lines of code",
```

Comment:
@m-g-sonar implementation does not yet support "lines of code", we have to keep "lines"